kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -274,6 +279,19 @@ public static ConfigDef newConfigDef() {
           "S3 Part Size"
       );
 
+      configDef.define(
+        S3_STORAGE_CLASS,
+        Type.STRING,
```

Comment:
Why not `Type.CLASS`?

Reply:
It seems to me that Type.CLASS was for ClassName and not a value of a specific class/enum

Reply:
Yes... Why not `StorageClass.class`?

Reply:
i tried to use Type.CLASS and modified code to accept it, but i could'nt handle correctly with Enum.
i keep the code in a string way, but i'm open to suggestion 👍